server/services/timelineService.ts

```diff
@@ -18,7 +18,10 @@ export default class TimelineService {
     }
   }
 
-  filterTimelineByActionPlanCreatedEvent = (timelineResponse: TimelineResponse) => {
-    return timelineResponse.events.filter((event: { eventType: string }) => event.eventType === 'ACTION_PLAN_CREATED')
+  filterTimelineByActionPlanCreatedOrUpdatedEvent = (timelineResponse: TimelineResponse) => {
```

Comment:
This could possibly be 2 functions, but felt they were under the same category of "Action plan" events so grouped them together.

Reply:
Happy for it to be one function 👍 , it's how I envisaged it in these early commits.
Though I think this service class might change pretty soon once I wire in the prison name lookup! 😁

Reply:
One thing I did note is that we dont have a unit test for this class 😢 
Dont worry for now, because as mentioned I'm about to change this class quite a lot, so I'll create the skeleton unit class at that time; but I might ask you to flesh out some of the tests